ui/components/multichain/network-list-item/network-list-item.js

```diff
@@ -136,4 +147,5 @@ NetworkListItem.propTypes = {
    * Represents if the network item should be keyboard selected
    */
   focus: PropTypes.bool,
+  isDeprecatedNetwork: PropTypes.bool,
```

Comment:
We should add doc comments here in the future.

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/21933/commits/30f9a976ec84ad9c31351a3b8717845af2d14717